src/test/scala/com/gu/apis/SlackApiChannels.scala

```diff
@@ -43,8 +41,12 @@ case class SlackApiChannels(channel: String) {
     }
   }
 
+  def jsonToString(js: JsValue): String = js.toString.replace("\"", "")
+
   def getJsonAttribute(js: JsValue, attribute: String): String = {
-    jsonToString(js \ attribute)
+    jsonToString((js \ attribute).get)
```

Comment:
I'm going to completely rework this class in the next PR probably, so going to leave this one for now